src/main/java/seedu/address/model/person/Person.java

```diff
@@ -130,6 +111,13 @@ public boolean isSamePerson(Person otherPerson) {
                 && otherPerson.getName().equals(getName());
     }
 
+    /**
```

Comment:
Hang on, isn't this the same has `isSamePerson`? I was under the impression that `isSameXXX` would define a form of equality that the class was supposedly unique by (e.g. name for `Person`, name + date for `Event`)